File: features/wads/src/main/java/com/walhalla/utils/AManagerI.java
package com.walhalla.utils;

import android.app.Activity;
import android.content.Context;

public interface AManagerI {

    interface RewardManagerCallback {
        void successResult7(int position);

        void errorShowAds();

        //void onResumeVideo();
    }

    void loadRewardAd(Context context);

    void showRewardAdBanner(Activity activity, int position, RewardManagerCallback callback);
}

File: features/wads/src/main/java/com/walhalla/utils/AManagerI.kt
package com.walhalla.utils

import android.app.Activity
import android.content.Context

interface AManagerI {
    interface RewardManagerCallback {
        fun successResult7(position: Int)

        fun errorShowAds() //void onResumeVideo();
    }

    fun loadRewardAd(context: Context)

    fun showRewardAdBanner(activity: Activity, position: Int, callback: RewardManagerCallback?)
}
